app/views/_cascade/formats/level/_masthead.erb

```diff
@@ -0,0 +1,240 @@
+                      <input class="slideSubtitle"
+                             type="hidden"
+                             value="<%= slideSubtitle.html_safe %>"/>
+
+                      <% if slidePhotoBy != '' %>
+                        <div class="photo-by">
+                          Photo by: <%= slidePhotoBy %>
+                        </div>
+                      <% end %>
+                    </li>
+                  <% end %>
+                </ul>
+              </div>
+            </div>
+          </div>
+        </div>
+      </div>
+
+      <div class="column header theme-bg-color">
+        <div class="aligned">
+          <h2 class="header"><%= header.html_safe %></h2>
+          <div class="subheader <%= subheaderClass %>">
+            <hr />
+            <h4><%= firstSubtitle.html_safe %></h4>
+          </div>
+
+          <% if firstSubtitle != '' %>
+
+          <% end %>
+        </div>
+      </div>
+    </div>
+  </div>
+
+<% elsif mastheadType == 'Slider' && displaySlider == 'Yes' %>
+  <!-- mastheadType: "Slider" -->
+  <%
+    # VTL Format: #set ( $slides = $_XPathTool.selectNodes($masthead, 'slider/slides/slide') )
+    slides = current_page.select_nodes('old/slider/slides/slide')
+  %>
+
+  <div class="hidden slideOptions">
+      <div class="autoRotate">On</div>
+  </div>
+
+  <div class="rounded-slider">
+    <div class="rotatorContainer">
+      <div class="flex-container">
+        <div class="flexslider">
+          <ul class="slides">
+            <% slides.each do |slide| %>
+              <%
+                slideHeader = slide.select_single_node_value('header')
+                slideSubtitle = slide.select_single_node_value('sub-title')
+                slideDescription = slide.select_single_node_value('description')
+                slideLinkUrl = slide.select_single_node_value('link/link')
+                slideInternalLinkPath = slide.select_single_node_value('link/internalLink/path')
+                slideFileLinkPath = slide.select_single_node_value('link/fileLink/path')
+                slideImagePath = slide.select_single_node_value('image/path')
+                slideImageAltText = slide.select_single_node_value('altText')
+                slideBackground = slide.select_single_node_value('text-background')
+                slideQuoteAuthor = slide.select_single_node_value('quote-author')
+                slideCaptionAlign = slide.select_single_node_value('align')
+
+                # Set slide href.
+                if slideInternalLinkPath != '/'
+                  slideCaptionHref = slideInternalLinkPath
+                elsif slideInternalLinkPath == '/' && slideLinkUrl != ''
+                  slideCaptionHref = slideLinkUrl
+                elsif slideInternalLinkPath == '/' && slideLinkUrl == '' \
+                  && slideFileLinkPath != '/'
+                  slideCaptionHref = slideFileLinkPath
+                else
+                  slideCaptionHref = ''
+                end
```

Comment:
Looking at this I totally understand why this is written the way it is if this is how our Velocity looks on Cascade and this is just replicating that behavior, but phew. Two-liner conditionals with 3 conditions stiiiiiiiinks 💩